app/views/users/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<ul>
+    <% @user.errors.full_messages.each do |error| %>
+      <li><%= error %></li>
+    <% end %>
+</ul>
+
+<%= form_for(@user) do |f| %>
+  User name: <%= f.text_field :username, placeholder: "Your user name here" %><br />
+  Email: <%= f.email_field :email, placeholder: "example@example.com"%><br />
+  Password: <%= f.password_field :password %><br />
+  <%= f.submit "" %>
+<% end %>
```

Comment:
Using the empty quotes(`""`) in the submit button leaves your submit button without a label.